pkgs/by-name/ht/htb-toolkit/package.nix

```diff
@@ -15,14 +16,14 @@
 
 rustPlatform.buildRustPackage {
   pname = "htb-toolkit";
-  version = "0-unstable-2025-03-15";
+  version = "0-unstable-2025-08-12";
 
   src = fetchFromGitHub {
     owner = "D3vil0p3r";
     repo = "htb-toolkit";
     # https://github.com/D3vil0p3r/htb-toolkit/issues/3
```

Comment:
Is this comment still relevant ?

Reply:
Yeah. I would keep it and in the future, if I find some time, I try to create a tag.